### DISCUSSION TITLE
next/image Performance issue

### DISCUSSION THREAD

**Discussion by MrksDllngr** [vercel/next.js#23858] [Help]
Hey everybody!

I am new to nextjs and I see some issues with the next/Image component:

I have some pictures on a strapi backend and fetch them with getStaticProps()

when I build and serve the project in production, I see that when I load the page, the Images take up to 6s to be loaded:
![Bildschirmfoto 2021-04-09 um 17 16 44](https://user-images.githubusercontent.com/76658365/114203682-ab987800-9958-11eb-9f28-6d121e9f6f4c.png)

I guess this is the time next/image takes to create an optimized version of the image and serve it.

The Original Images in the graph above are quite big (around 6MB JPG)

when I try this with a smaller original, it looks a lot better:
![Bildschirmfoto 2021-04-09 um 17 21 35](https://user-images.githubusercontent.com/76658365/114203939-ed292300-9958-11eb-8405-b1f5fcd1fe95.png)

When it was loaded first, it seems that it is a lot faster when I do a hard refresh of the page (takes around 5ms)
After a bit of time it is again very slow...

I found this on the official documentation on the Nextjs Homepage:

> Images are optimized dynamically upon request and stored in the <distDir>/cache/images directory. The optimized image file will be served for subsequent requests until the expiration is reached. When a request is made that matches a cached but expired file, the cached file is deleted before generating a new optimized image and caching the new file.
> 
> The expiration (or rather Max Age) is defined by the upstream server's Cache-Control header.
> 
> If s-maxage is found in Cache-Control, it is used. If no s-maxage is found, then max-age is used. If no max-age is found, then 60 seconds is used.

Is that the explanation of that behaviour?
Can I just set the expiration time to 100days and everything is fine?
The Images will not change on a regular basis, so this should be long enough for my use case

How do you get by with these issue?
is this known?

Regards Markus

**Comment by MrksDllngr** [comment]
Anybody who knows an answer?

**Comment by JimmyGray** [comment]
@MrksDllngr I'm experiencing the same performance problems. In my case it appears to be the following issue: 
https://github.com/vercel/next.js/issues/23637

**Comment by ErnestBrandi** [comment]
Also be aware there is currently problem setting cache control options yourself : https://github.com/vercel/next.js/issues/19914
